_includes/setup/ios-setup.md

```diff
@@ -1,8 +1,5 @@
 ## iOS setup
 
-_This step is optional, and can be performed at any time before
```

Comment:
This is technically optional, though, yes? I can use my mac to develop android apps with Flutter.